call-home/Cargo.toml

```diff
@@ -20,8 +20,8 @@ path = "src/bin/stats/main.rs"
 [dependencies]
 constants = { path = "../constants" }
 openapi = { path = "../dependencies/control-plane/openapi" }
-kube = { version = "0.87.0", features = ["runtime", "derive"] }
-k8s-openapi = { version = "0.20.0", features = ["v1_22"] }
+kube = { version = "0.94.2", features = ["runtime", "derive"] }
```

Comment:
This will need changes on openebs/openebs repo also right? We use this dependency there also.

Reply:
We might not use call-home there. So should be okay i guess.

Reply:
Yes, we use this as a dependency. The outdated dependencies cause compilation issues.